framework/DataObjects/TestXDataSet.py

```diff
@@ -278,7 +278,7 @@ def formatRealization(rlz):
 # NOTE histories are currently disabled pending future work (c,y are history vars)
 checkArray('DataSet __init__ inp',data._inputs,['a','b','c'],str)
 checkArray('DataSet __init__ out',data._outputs,['x','y','z'],str)
-checkArray('DataSet __init__ all',data._allvars,['a','b','c','x','y','z'],str)
+checkArray('DataSet __init__ all',data.vars,['a','b','c','x','y','z'],str)
```

Comment:
Is it ```orderedVars```

Reply:
No, since we're not checking pivot parameters that are misaligned.  `data.vars` returns the non-index variables.